webui/react/src/pages/TrialDetails.tsx

```diff
@@ -132,7 +132,7 @@ const TrialDetailsComp: React.FC = () => {
       <TrialActions trial={trial.data}
         onClick={handleActionClick}
         onSettled={pollTrialDetails} />
-      <Section title="Info Box">
+      <Section bodyBorder title="Info Box">
```

Comment:
IMO usage of `Section` makes more sense in a page component as opposed to internal components eg the current experimentChart or experimentInfoBox. thoughts/?

Reply:
excellent call

Reply:
I just remembered that it was used in the internal components because of the section `options` props. So for example, the linear vs log dropdown was easier to handle inside the `ExperimentChart` internal component, rather than wrapping the `ExperimentChart` with `Section` and handling all the filter/control logic outside of the component.

Reply:
makes sense. :+1: